cmd/nfd-topology-updater/main.go

```diff
@@ -112,12 +112,13 @@ func parseArgs(flags *flag.FlagSet, osArgs ...string) (*topology.Args, *resource
 	}
 
 	if len(resourcemonitorArgs.KubeletConfigURI) == 0 {
-		if len(utils.NodeName()) == 0 {
-			fmt.Fprintf(flags.Output(), "unable to determine the default kubelet config endpoint 'https://${NODE_NAME}:%d/configz' due to empty NODE_NAME environment, "+
-				"please either define the NODE_NAME environment variable or specify endpoint with the -kubelet-config-uri flag\n", kubeletSecurePort)
+		nodeIP := os.Getenv("NODE_IP")
```

Comment:
you can use `LookupEnv` here https://pkg.go.dev/os#LookupEnv
```go

Reply:
do we care about checking if `NODE_IP` is a valid ipv4 or ipv6 address?

Reply:
I'm not sure. Now that we think about it we could name it NODE_ADDRESS or something instead so user could put whatever, IP, nodename, hostname... WDYT?

Reply:
sounds good :)

Reply:
If the env var is not defined `os.GetEnv` will return an empty string -> the error case is handled below by `if len(nodeIP) == 0`